### PR TITLE
Grammar homework is done

### PR DESCRIPTION
**Pull request by breackinumber**
My grammar home task has been done and i need to pull it to the main branch.

### CODE REVIEW

File: homework/english/semenenko_artur/lesson1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Let´s go out. It is raining  now.
+
+I have to go now. It is getting  dark.
+
+A young man usually stands  at the window.
+
+Ann usually makes doll dresses herself.
+
+ Is his wife having  coffee at the moment?
```
Comment (VolhaZh) [inline homework/english/semenenko_artur/lesson1/grammar.md:27]: i thought "at the moment" is dont meaning "now"
so it seems to need simple prasent form

**Review by VolhaZh — COMMENTED** [review]
lgfm

File: homework/english/semenenko_artur/lesson1/grammar.md (hunk 1 above)
Comment (goldenas18) [reply homework/english/semenenko_artur/lesson1/grammar.md:27]: but shes still enjoying it)

**Review by goldenas18 — APPROVED** [review]

File: homework/english/semenenko_artur/lesson1/grammar.md (hunk 1 above)
Comment (breackinumber) [reply homework/english/semenenko_artur/lesson1/grammar.md:27]: VolhaZh, i'm sure it is not a mistake. "to be" + V1. And she having coffee right now.

File: homework/english/semenenko_artur/lesson1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+
+
+Is your mum working today?
+
+Why are these boys talking now?
```
[the remark this answers is not in the document]
Comment (breackinumber) [reply homework/english/semenenko_artur/lesson1/grammar.md:11]: **these**, because boy**s** is plural form of _boy_

**Review by goldenas18 — APPROVED** [review]

**Review by crptbosyak — APPROVED** [review]

File: homework/english/semenenko_artur/lesson1/grammar.md (hunk 1 above)
Comment (VolhaZh) [reply homework/english/semenenko_artur/lesson1/grammar.md:27]: ok

**Review by VolhaZh — APPROVED** [review]

**Pull request merged**